app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+  before_action :redirect_users_to_home, only: [:new, :create]
+
+  def create
+    user = User.find_by_email(params[:email])
```

Comment:
Tomas, do you have any source information that it's better to use find_by(property: value) instead of find_by_property(value) ? I couldn't find it with a quick search.
Personally I would vouch for using find_by_property as it's more readable.